companies/arcteryx.md

```diff
@@ -0,0 +1,25 @@
+---
+fortune_rank: number
+name: Arc'teryx
+tags: ["fortune500", "sports"]
+image: arcteryx.jpeg
+blm_statements:
+  - url: https://www.instagram.com/p/CA5v5wYJsb0/
+    date_posted: 2020-06-01
+    raw_text: We have stayed silent too long. 
```

Comment:
I get an error when I run this locally due to indention :( 

![image](https://user-images.githubusercontent.com/21069635/97241827-3842e580-17c0-11eb-840f-1e7777ed2462.png)

For a multi-line statement, you can use:
`raw_text: >` and then ident the rest of the statement inward.

The [Apple data file](https://raw.githubusercontent.com/DiversityCorp/companies-on-blm/main/companies/apple.md) is a good example of how to do this.  We are also currently working on automating catching these errors so it will be easier to find :)